symphony/bdk/core/config/model/bdk_ssl_config.py

```diff
@@ -4,17 +4,14 @@
 
 
 class BdkSslConfig:
-    """Class containing the SSL configuration.
-    """
+    """Class containing the SSL configuration."""
 
     def __init__(self, config):
         """
 
         :param config: the dict containing the SSL specific configuration.
         """
         if config is not None and TRUST_STORE in config:
-            truststore_config = config.get(TRUST_STORE)
-            self.trust_store = BdkCertificateConfig(path=truststore_config.get("path"),
-                                                    password=truststore_config.get("password"))
+            self.trust_store = BdkCertificateConfig(path=config[TRUST_STORE].get("path"))
```

Comment:
we don't have a password anymore?

Reply:
in the java we also support passing a content (instead of a path) do we want to do it here too? (at least to support the same config files?)

Reply:
No need for password: https://docs.python.org/3/library/ssl.html#ssl.SSLContext.load_verify_locations 
Let me add content support.